test/testdata/container_sleep.json

```diff
@@ -0,0 +1,37 @@
+{
+	"metadata": {
+		"name": "podsandbox-sleep"
+	},
+	"image": {
+		"image": "quay.io/crio/redis:alpine"
```

Comment:
for some reason the CI test fails to find this image.

Would it be possible for `test/shm.bats` to re-use `container_config_sleep.json` that is already present under testdata or is there anything different here (the only difference I see is the privileged flag)?

Alternatively, could `quay.io/crio/busybox:latest` be used if we are just using sleep?

Reply:
I actually already fixed this but I pushed a new PR.